### PR TITLE
Reviews Page

### PR DESCRIPTION
**Pull request by Raemon**
I found myself wanting a page where I could a) look at reviews without having to load all the voting stuff, b) could see all reviews from all years.

image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203669798230310) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/routes.ts [hunk 1]
```diff
@@ -920,23 +920,23 @@ const forumSpecificRoutes = forumSelect<Route[]>({
...
-      path: '/reviews/2018',
-      componentName: 'Reviews2018',
-      title: "2018 Reviews",
-    },
+    // {
+    //   name: 'reviews2018',
+    //   path: '/reviews/2018',
+    //   componentName: 'Reviews2018',
+    //   title: "2018 Reviews",
+    // },
```
Comment (jpaddison3) [inline packages/lesswrong/lib/routes.ts:928]:
```suggestion
```

File: packages/lesswrong/lib/routes.ts [hunk 2]
```diff
@@ -920,23 +920,23 @@ const forumSpecificRoutes = forumSelect<Route[]>({
...
-      path: '/reviews/2019',
-      componentName: 'Reviews2019',
-      title: "2019 Reviews",
-    },
+    // {
+    //   name: 'reviews2019',
+    //   path: '/reviews/2019',
+    //   componentName: 'Reviews2019',
+    //   title: "2019 Reviews",
+    // },
```
Comment (jpaddison3) [inline packages/lesswrong/lib/routes.ts:939]:
```suggestion
```

**Review by jpaddison3 — COMMENTED** [review]
No objections from the EA Forum

**Comment by Raemon** [comment]
I'm kinda on my own this week. This PR doesn't affect things outside of the Review pages. I'm not 100% sure about the refactoring I did here but it's easy to undo or change if it turns out to have been overzealous.

Merging.

**Pull request merged**